com.unity.render-pipelines.core/Runtime/Textures/RTHandleSystem.cs

```diff
@@ -217,23 +222,37 @@ public void SetReferenceSize(int width, int height, MSAASamples msaaSamples, boo
                 lastFrameMaxSize = new Vector2(GetMaxWidth(), GetMaxHeight());
             }
 
+            var scales = CalculateRatioAgainstMaxSize(m_RTHandleProperties.currentViewportSize);
```

Comment:
Ping @FrancescoC-unity 
What do you think of this refactor? I think is more elegant to just divide times the prescaled max.
This way you preserve the correct rations when you establish a smaller reference size. Let me know.

Reply:
I think we discussed this, it's been 2 months now and I forgot :D

Reply:
Yes, this was done to preserve correct ratios between destination. What you want, always is a scaled max, a scaled current resolution and compute the ratio of that, because of rounding (a texel of rounding in the smaller res means a lot more texels at full res).
This means that if we dont do this, the aspect ratio at smaller resolutions wabbles. This is only noticible in resolutions < %30 scales. I do think is important to have becasue DLSS does pretty well that low so its likely users will be pushing resolution down a lot.